sjk-core/src/test/java/org/gridkit/jvmtool/cmd/StackSampleAnalyzerCmdTest.java

```diff
@@ -0,0 +1,49 @@
+package org.gridkit.jvmtool.cmd;
+
+import org.gridkit.jvmtool.cmd.StackSampleAnalyzerCmd.DateFormater;
+import org.gridkit.jvmtool.cmd.StackSampleAnalyzerCmd.DecimalFormater;
+import org.gridkit.jvmtool.cmd.StackSampleAnalyzerCmd.MemRateFormater;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.util.TimeZone;
+
+public class StackSampleAnalyzerCmdTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  @Test
+  public void testToStringDateFormater() {
```

Comment:
To my mind, `testDateFormaterToString` would make more sense as the method name format (like `DateFormater.toString` but without the dots).

Reply:
Agreed, thanks. Resolved this.